cadl-tests/src/main/java/com/cadl/longrunning/LongRunningClient.java

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.cadl.longrunning;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.exception.ClientAuthenticationException;
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.exception.ResourceModifiedException;
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.SimpleResponse;
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.Context;
+import com.azure.core.util.polling.SyncPoller;
+import com.cadl.longrunning.models.Resource;
+
+/** Initializes a new instance of the synchronous LongRunningClient type. */
+@ServiceClient(builder = LongRunningClientBuilder.class)
+public final class LongRunningClient {
+    @Generated private final LongRunningAsyncClient client;
+
+    /**
+     * Initializes an instance of LongRunningClient class.
+     *
+     * @param client the async client.
+     */
+    @Generated
+    LongRunningClient(LongRunningAsyncClient client) {
+        this.client = client;
+    }
+
+    /**
+     * Creates or updates a Resource asynchronously.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * {
+     *     id: String (Required)
+     *     name: String (Required)
+     *     type: String (Required)
+     * }
+     * }</pre>
+     *
+     * @param name The name parameter.
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @return the {@link SyncPoller} for polling of long-running operation.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
+    public SyncPoller<BinaryData, BinaryData> beginCreateOrUpdate(String name, RequestOptions requestOptions) {
+        return this.client.beginCreateOrUpdate(name, requestOptions).getSyncPoller();
+    }
```

Comment:
@haolingdong-msft @XiaofeiCao 

When you check the deviceupdate LRO in create, does intermediate type and final type same?

Reply:
From the provided session-record for `testImportUpdates`, I believe they are the same.

Reply:
Emm, OK, thanks. "import" might be different from CRUD (similar to the "importx" and "export" in authoring, as action).

Reply:
For DPG it is simple, as both are `BInaryData`.

It would get complicated when we generate convenience method.